### ISSUE TITLE
New "What We Do" illustrations

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#739]
Our current [What We Do](https://cloudfour.com/does/) page content lacks illustrations for design systems or accessibility. We should create those.

---

- See #725 
- See #738

**Comment by tylersticka** [comment]
@dromo77 Can you summarize what the state of this is? If it isn't actively in progress, can you un-assign yourself and move it to "Up Next"? Thank you!

**Comment by tylersticka** [comment]
@dromo77 Were there any assets created for this or was it never started?

**Comment by dromo77** [comment]
@tylersticka I never started working on this.

**Comment by Paul-Hebert** [comment]
I created new a11y and design system icons. I also took a pass at standardizing all of the icons so they're optical sizing is more consistent. The output of this task can be found in Drive: https://drive.google.com/drive/u/0/folders/1j4YjdWb2rSHSn-MqzjEJgR1U2G0YUlGp
